### PR TITLE
Change to use Terra.describeViewports part 2

### PR DESCRIPTION
**Pull request by Blackop778**
### Summary
Also updating to use `Terra.it.validatesElement`

### CODE REVIEW

File: packages/terra-button-group/tests/wdio/button-group-spec.js [hunk 1]
```diff
@@ -1,61 +1,59 @@
...
       browser.moveToObject('#root', 0, 900); // move cursor to remove focus styles
     });
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot({ selector: '#button-group-text' });
+
+    Terra.it.validatesElement({ selector: '#button-group-text' });
   });
 
-  describe('Button Group Wrapping', () => {
+  Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
```
Comment (emilyrohrbough) [inline packages/terra-button-group/tests/wdio/button-group-spec.js]: This should still be a Mocha `describe`

File: packages/terra-button-group/tests/wdio/button-group-spec.js [hunk 2]
```diff
@@ -1,61 +1,59 @@
...
       browser.moveToObject('#root', 0, 900); // move cursor to remove focus styles
     });
-    Terra.it.isAccessible({ viewports });
-    Terra.it.matchesScreenshot({ viewports });
+
+    Terra.it.validatesElement();
   });
 
-  describe('Button Group Block', () => {
+  Terra.describeViewports('Button Group Block', ['tiny', 'huge'], () => {
```
Comment (emilyrohrbough) [inline packages/terra-button-group/tests/wdio/button-group-spec.js]: This should still be a Mocha `describe`

File: packages/terra-button-group/tests/wdio/button-group-spec.js (hunk 1 above)
Comment (Blackop778) [reply packages/terra-button-group/tests/wdio/button-group-spec.js]: You think all the tests should be ran against tiny and not just these select few? Previously it was only these two groups of tests ran against tiny.

Comment (StephenEsser) [reply packages/terra-button-group/tests/wdio/button-group-spec.js]: I'm assuming Terra.describeViewports should never be nested, if you want to test different viewports for the same spec file they need to be split out as siblings.

```jsx
Terra.describeViewports('', () => {
});

Terra.describeViewports('', () => {
})
```

Comment (Blackop778) [reply packages/terra-button-group/tests/wdio/button-group-spec.js]: Fixed in 516460b5681319749429cae1b796e1ffa03f0b1f

File: packages/terra-button-group/tests/wdio/button-group-spec.js (hunk 2 above)
Comment (Blackop778) [reply packages/terra-button-group/tests/wdio/button-group-spec.js]: Fixed in 516460b5681319749429cae1b796e1ffa03f0b1f

File: packages/terra-button-group/tests/wdio/button-group-spec.js [hunk 3]
```diff
@@ -97,41 +74,54 @@ describe('Button Group', () => {
...
       expect(browser.getText('#selected-keys')).to.equal('2');
     });
 
-    Terra.it.isAccessible();
-    Terra.it.matchesScreenshot('button3-unselected');
+    Terra.it.validatesElement('button3-unselected');
+  });
+});
+
+Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
```
Comment (emilyrohrbough) [inline packages/terra-button-group/tests/wdio/button-group-spec.js]: To reduce the number of viewport changes during the test run, we will want to simplify this:

```
Terra.describeViewports('Button Group Wrapping', ['tiny', 'huge'], () => {
    describe('Button Group Wrapping', () => {
         // the test
    });

    describe('Button Group Block', () => {
         // the test
    });
});
```

Comment (Blackop778) [reply packages/terra-button-group/tests/wdio/button-group-spec.js]: Changed in 57c132c360d0472bd6b073954b90e0144db53946

**Pull request merged**